capable/ffi/sys/build.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! Builds the FFI type bindings for the types used by libsgx_capable.{so,a}.
```

Comment:
I think we said the `so` wasn't going to be used
```suggestion
//! Builds the FFI type bindings for the types used by libsgx_capable.a.
```